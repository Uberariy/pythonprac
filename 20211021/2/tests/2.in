:cos x cos(x)
cos 0.5
cos 1
cos 1.5
cos 2
cos 2.5
quit cos
