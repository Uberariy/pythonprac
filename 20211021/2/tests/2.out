5.0
10
15.0
20
25.0
2:uuu7
